<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplifiedSearch</class>
 <widget class="QDialog" name="SimplifiedSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易查询</string>
  </property>
  <widget class="QCheckBox" name="_checkBoxUnLabelled">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>未标注图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_buttonSearch">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_buttonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_comboBoxPlaneID">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>104</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>111</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>机型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_comboBoxPlaneType">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>105</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>53</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>单位</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_comboBoxUnit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
